include/z_en_hy_code.h

```diff
@@ -57,10 +57,10 @@ typedef struct EnHy {
     /* 0x205 */ u8 isRightFootOnGround;
     /* 0x206 */ Vec3s jointTable[ENHY_LIMB_MAX];
     /* 0x266 */ Vec3s morphTable[ENHY_LIMB_MAX];
-    /* 0x2C6 */ Vec3s turnTarget;
+    /* 0x2C6 */ Vec3s trackTarget;
     /* 0x2CC */ Vec3s headRot;
     /* 0x2D2 */ Vec3s torsoRot;
-    /* 0x2D8 */ Vec3s tmpTurnTarget;
+    /* 0x2D8 */ Vec3s tmptrackTarget;
     /* 0x2DE */ Vec3s tmpHeadRot;
     /* 0x2E4 */ Vec3s tmpTorsoRot;
```

Comment:
This is probably related to a different `SubS` PR I missed, but what are these `tmp` prefixes for?

Reply:
Ah, those don't really have anything to do with `SubS`. Instead the `EnHy` system has space to backup the three variables above without the `tmp` prefix, and restore them back (same with `tmpActionFunc`). `EnCne01` and `EnBba01` uses it save these three variables just before talking, and restore them back after finishing talking. Probably should rename those better (kind of thinking suffixing `backup`), but would prefer to do that in a different PR as I've used the same terminology in other places that I would also like to change.

Reply:
Ah fair, I believe the prefix `prev` is fairly common in OoT/MM for that kind of stored temp with the intent to restore. But that can be a separate PR yeah